pages/02.Documentation/07.images/default.en.md

```diff
@@ -1,45 +1,49 @@
 ---
 title: Images
 published: true
-date: '08-08-2018 11:06'
+date: '17-08-2018 16:22'
 taxonomy:
     category:
         - docu
 ---
 
 ## Overview
 
-SysEleven provides and maintains a set of images in the SysEleven Stack.  
+SysEleven provides and maintains a set of images in the SysEleven Stack.
 The SysEleven Stack also offers the possibility to upload custom images.
 
 You can view and manage images both via the OpenStack API, as well as using the [Dashboard (GUI)](https://dashboard.cloud.syseleven.net).
 
-## Provided Images
+## Available Images
 
-Images with the description **sys11 optimized** are built, optimized, tested and fully supported by SysEleven.  
-Standard cloud images from their original source are supported with best effort only.
+### SysEleven Images
 
-Distro           | Name                             | Description                               | Supported   |
------------------|----------------------------------|-------------------------------------------|-------------|
-Ubuntu 18.04 LTS | Ubuntu 18.04 LTS                 | Unmodified, directly from Canonical       | best-effort |
-Ubuntu 18.04 LTS | Ubuntu 18.04 LTS sys11 optimized | Modified, optimized for SysEleven Stack   | yes         |
-Ubuntu 16.04 LTS | Ubuntu 16.04 LTS                 | Unmodified, directly from Canonical       | best-effort |
-Ubuntu 16.04 LTS | Ubuntu 16.04 LTS sys11 optimized | Modified, optimized for SysEleven Stack   | yes         |
-Ubuntu 14.04 LTS | Ubuntu 14.04 LTS                 | Unmodified, directly from Canonical       | best-effort |
-Ubuntu 14.04 LTS | Ubuntu 14.04 LTS sys11 optimized | Modified, optimized for SysEleven Stack   | yes         |
-Ubuntu 16.04 LTS | Rescue Ubuntu 16.04 sys11        | Modified, optimized for SysEleven Stack   | yes         |
-Ubuntu 18.04 LTS | Rescue Ubuntu 18.04 sys11        | Modified, optimized for SysEleven Stack   | yes         |
+Images with the description **sys11 optimized** are built, optimized, tested and fully supported by SysEleven.
```

Comment:
remove "fully" along with "only" that has already been removed from the opposite statement